recipe/bld.bat

```diff
@@ -1,17 +1,22 @@
+@echo on
+%PYTHON% %RECIPE_DIR%\builder.py
+if errorlevel 1 exit 1
 
-mkdir build && cd build
+REM previous script for posterity
 
-cmake -LAH -G"NMake Makefiles"                ^
-  -DCMAKE_BUILD_TYPE=Release                  ^
-  -DCMAKE_PREFIX_PATH="%LIBRARY_PREFIX%"      ^
-  -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%"   ^
-  -DDLIB_LINK_WITH_SQLITE3=OFF                ^
-  -DDLIB_FORCE_MSVC_STATIC_RUNTIME=OFF        ^
-  ../tools/python
-if errorlevel 1 exit 1
+REM mkdir build && cd build
 
-cmake --build .
-if errorlevel 1 exit 1
+REM cmake -LAH -G"NMake Makefiles"                ^
```

Comment:
This is all in the git history so let's just remove it

Reply:
42d9523